### ISSUE TITLE
SLU16 Erroneous assumption on data set input to fit method ex.4 RandomizedSearchCV

### ISSUE THREAD

**Issue by orlandocosta76** [LDSSA/bootcamp#56]
On exercise 4, it is asked to "Adjust the KNN random search to the data" but it is not said that we should use the whole airbnb data set instead if the usual `X_train`, `y_train `resulting from sklearn's `train_test_split`.

The learning notebook use the `train_test_split` approach, and explains why (we need the test set to make decisions on which model to use, by using cross_validate, etc.), thus it is not consistent with the exercise notebook (for ex.2 with GridSearch the assert used has a wide interval that it makes irrelevant if we use that approach or not).

A minor issue is on ex.3, it should be clear which weights to use in the search parameter space, as the text:
_distante weight uniform or weight_
seems to point we should use both `uniform` and `distance `

**Comment by Juliana-R** [comment]
**Issue inactive for more than 1 year**, will be closed. Reopen if necessary.
